polaris-service/src/main/java/org/apache/polaris/service/context/PolarisCallContextCatalogFactory.java

```diff
@@ -82,8 +82,6 @@ public Catalog createCallContextCatalog(
             taskExecutor,
             fileIOFactory);
 
-    context.contextVariables().put(CallContext.REQUEST_PATH_CATALOG_INSTANCE_KEY, catalogInstance);
```

Comment:
Curious how the catalogInstance is set in `context` if we remove this line.

Reply:
The call context is re-created once this method exits:

https://github.com/apache/polaris/blob/d854a371c7d557c87801458be752953d053cd268/polaris-service/src/main/java/org/apache/polaris/service/catalog/PolarisCatalogHandlerWrapper.java#L145-L152